pkg/archive/changes.go

```diff
@@ -359,7 +359,7 @@ func ChangesSize(newDir string, changes []Change) int64 {
 }
 
 // ExportChanges produces an Archive from the provided changes, relative to dir.
-func ExportChanges(dir string, changes []Change) (Archive, error) {
+func ExportChanges(dir string, changes []Change, excludes []string) (Archive, error) {
```

Comment:
perhaps lets add a function of `ExportChangesExcludes` or similar. It is the best effort of the `./pkg/` to not break api for folks that may be using it. It's cool if you refactor this and the new function to reuse logic where possible.